cassandane/Makefile

```
@@ -44,8 +44,6 @@ PERL := perl
 all clean install::
 	@for dir in $(SUBDIRS) ; do cd $$dir ; $(MAKE) $@ || exit 1 ; done
 
```

Since the "all" target no longer appears twice, this rule above no longer needs the [double-colon](https://www.gnu.org/software/make/manual/html_node/Double_002dColon.html)

```suggestion
all clean install:
	@for dir in $(SUBDIRS) ; do cd $$dir ; $(MAKE) $@ || exit 1 ; done

```